### PR TITLE
Eagerly load TestEvent.results

### PR DESCRIPTION
**Pull request by emmastephenson**
# BACKEND PULL REQUEST

## Related Issue

- Responding to comments from #3788.

## Changes Proposed

- Changes the loading type from lazy -> eager for the ResultSet on TestEvent. Any time we're looking at a TestEvent we'll want the results, so there's no need for this to be a lazy implementation.

## Additional Information

- Tested locally with the Results page, conduct tests, and PXP; this change should be a no-op.

## Checklist for Primary Reviewer

- [ ] Any large-scale changes have been deployed to `test`, `dev`, or `pentest` and smoke tested
- [ ] Any content updates (user-facing error messages, etc) have been approved by content team
- [ ] Any changes that might generate questions in the support inbox have been flagged to the support team
- [ ] GraphQL schema changes are backward compatible with older version of the front-end
- [ ] Changes comply with the SimpleReport Style Guide
- [ ] Changes with security implications have been approved by a security engineer (changes to  authentication, encryption, handling of PII, etc.)
- [ ] Any dependencies introduced have been vetted and discussed

### CODE REVIEW

**Review by zdeveloper — DISMISSED** [review]
LGTM! thanks for adding this change

**Review by zdeveloper — DISMISSED** [review]
LGTM! has there been any measurable effect on performance?

File: backend/src/main/java/gov/cdc/usds/simplereport/db/model/TestEvent.java [hunk 1]
```diff
@@ -46,7 +46,7 @@ public class TestEvent extends BaseTestInfo {
   private TestOrder order;
 
   @JsonIgnore
-  @OneToMany(mappedBy = "testEvent", cascade = CascadeType.MERGE, fetch = FetchType.LAZY)
+  @OneToMany(mappedBy = "testEvent", cascade = CascadeType.MERGE)
```
Comment (mehansen) [inline backend/src/main/java/gov/cdc/usds/simplereport/db/model/TestEvent.java:49]: Is the default for fetch EAGER?
In [this doc](https://docs.oracle.com/javaee/6/api/javax/persistence/OneToMany.html) it looks like the default is to lazy load, but I'm not sure if I'm looking in the right place.

Comment (emmastephenson) [reply backend/src/main/java/gov/cdc/usds/simplereport/db/model/TestEvent.java]: Good catch, you're right - the default is lazy, so I changed to an explicit eager. Thanks!

**Comment by emmastephenson** [comment]
@zdeveloper I remembered why I didn't eagerly load results in the first place 😅 If you do, the n plus one tests start failing for fetching test events. I don't know of any easy ways around that, so I might close this PR without merging unless you have other suggestions.

**Pull request closed without merging**